xml/deploy_cephadm.xml

```diff
@@ -5,22 +5,107 @@
     %entities;
 ]>
 <chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="deploy-cephadm">
- <title>Deploying with the cephadm Module</title>
+ <title>Deploying with &cephadm;</title>
  <info>
   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
    <dm:editurl>https://github.com/SUSE/doc-ses/edit/master/xml/</dm:editurl>
    <dm:translation>yes</dm:translation>
    <dm:release>SES 7</dm:release>
   </dm:docmanager>
  </info>
+ <para>
+  To deploy a &ceph; cluster by using the &cephadm;, you need to achieve the
+  following tasks:
+ </para>
+ <orderedlist>
+  <listitem>
+   <para>
+    Install and do basic configuration of the underlying operating
+    system&mdash;&cephos;&mdash;on all cluster nodes.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Deploy the &salt; infrastructure over all cluster nodes so that you can
+    orchestrate them effectively.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Configure basic properties of the cluster and deploy it.
+   </para>
+  </listitem>
+ </orderedlist>
  <sect1 xml:id="deploy-os">
-  <title>Install &cephos;</title>
+  <title>Install and Configure SLES</title>
 
-  <para></para>
+  <procedure>
+   <step>
+    <para>
+     Install and register &cephos; on each cluster node. Include the following
```

Comment:
I think we should also specify that they should do a "minimal" installation  -- no X etc. -- and that using JeOS is an option. What do you think?

Reply:
No, JeOS never was a requirement, let's wait what SES team comes with